Sources/Swifter/DemoServer.swift

```diff
@@ -17,7 +17,12 @@ public func demoServer(publicDir: String?) -> HttpServer {
 
     server["/"] = { r in
         var listPage = "Available services:<br><ul>"
-        listPage += server.routes.map({ "<li><a href=\"\($0)\">\($0)</a></li>"}).joinWithSeparator("")
+		for route in server.routes {
```

Comment:
Oh, I didn't even notice that this project uses spaces for indentation. I have just always used tabs and most projects I've worked on have as well. I'll update the pull request with spaces for indentation later